internal/controller/suite_test.go

```diff
@@ -205,22 +204,13 @@ func downloadCRD(url string, downloadDir string) {
 
 func getNumaflowControllerDefinitions() config.NumaflowControllerDefinitionConfig {
 	// Read definitions config file
-	// TODO: use this file instead "../../tests/config/controller-definitions-config.yaml"
-	data, err := os.ReadFile("../../config/manager/numaflow-controller-definitions-config.yaml")
+	configData, err := os.ReadFile("../../tests/config/controller-definitions-config.yaml")
```

Comment:
Using this file in unit test, so that I can have at least two version of numaflow for testing

Reply:
I was thinking we could use `../../tests/config/controller-definitions-config.yaml` for more test cases and add more versions to it. Some versions could be made in a way that causes errors so we can also test "unhappy" cases. Thank you for migrating from `numaflow-controller-definitions-config.yaml` to this test file.